cfgov/v1/views/reports.py

```diff
@@ -106,3 +112,44 @@ def get_filename(self):
 
     def get_queryset(self):
         return Document.objects.all().order_by("-id").prefetch_related("tags")
+
+
+class ImagesReportView(ReportView):
+    header_icon = "image"
+    title = "All images"
+
+    list_export = [
+        "title",
+        "file",
+        "file_size",
+        "collection",
+        "tags.names",
+        "created_at",
+        "uploaded_by_user",
+    ]
+    export_headings = {
+        "title": "Title",
+        "file": "File",
+        "file_size": "Size",
+        "collection": "Collection",
+        "tags.names": "Tags",
+        "created_at": "Uploaded on",
+        "uploaded_by_user": "Uploaded by",
+    }
+
+    custom_field_preprocess = {
+        "tags.names": {"csv": process_tags},
+        "file": {"csv": construct_files_url},
+    }
+
+    template_name = "v1/images_report.html"
+
+    def get_filename(self):
+        """Get a better filename than the default 'spreadsheet-export'."""
+        return f"all-cfgov-images-{date.today()}"
+
+    def get_queryset(self):
+        raw_set = Image().objects.all().order_by("-created_at").prefetch_related("tags")
+        for image in raw_set:
+            image.url = construct_files_url(image.file)
+        return raw_set
```

Comment:
I think you'll need a blank line here at the end of the file to satisfy the linters.